src/main/java/com/google/sps/data/Availability.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+import java.time.LocalDate;
+
+/**
+ * Availability is a time range when a given person is available to offer interviews as an
+ * interviewer.
+ */
+@AutoValue
+public abstract class Availability {
+  abstract String email();
+
+  abstract TimeRange when();
+
+  abstract LocalDate date();
+
+  public static Availability create(String email, TimeRange when, LocalDate date) {
+    return builder().setEmail(email).setWhen(when).setDate(date).build();
```

Comment:
Nit: Break these into multiple lines so that people can add new fields later.

Reply:
l can but the formatter just zaps them back up again. 
Honestly l think l'll probably forget to change it back to this multiple line format every time l do a PR, and l apologize in advance. But the good news is that when people add new fields later the line will get so long that the formatter will divide them into new lines.

Reply:
What formatter are you using?